svc-api/rpc/chassis.go

```diff
@@ -113,3 +112,20 @@ func UpdateChassis(req chassisproto.UpdateChassisRequest) (*chassisproto.GetChas
 	}
 	return resp, nil
 }
+
+//UpdateChassisResource will do the rpc call to update a chassis resource
+func UpdateChassisResource(req chassisproto.UpdateChassisResourceRequest) (*chassisproto.GetChassisResponse, error) {
+	conn, err := services.ODIMService.Client(services.Systems)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to create client connection: %v", err)
```

Comment:
Use the same error description as error below - "error: failed to create client connection" or remove "error:" from error below

Reply:
Removed "error:" from error below.